src/main/java/no/nav/familie/inntektsmelding/forespørsel/rest/ForespørselRest.java

```diff
@@ -64,7 +64,7 @@ public class ForespørselRest {
     @Path("/oppdater")
     @Tilgangskontrollert
     public Response oppdaterForespørsler(@Valid @NotNull OppdaterForespørslerRequest request) {
-        LOG.info("Mottok forespørsel om oppdatering av inntektsmeldingoppgaver på fagsakSaksnummer {}", request.fagsakSaksnummer());
+        LOG.info("Mottok forespørsel om oppdatering av inntektsmeldingoppgaver på saksnummer {}", request.fagsakSaksnummer());
```

Comment:
Skal vi endre requesten og? 
```suggestion
        LOG.info("Mottok forespørsel om oppdatering av inntektsmeldingoppgaver på saksnummer {}", request.saksnummer());
```

Reply:
Helst, men det er litt tuklete siden k9-sak er avh. av dette. Må lage et nytt felt og deprekere det gamle tror jeg.